### PR TITLE
Command proposals

### PR DESCRIPTION
**Pull request by tomasmik**
Example of the new command:

```
tomas@pop-os:~/go/src/github.com/mysteriumnetwork/node$ build/myst/myst proposals --location-type hosting --country GB
[INFO] Found proposals:
| Identity: 0xb022800e11233a963040a48f456de922f2d0cc5d | Type: hosting | Country: GB | Price: 0.000010MYST/min   0.100000MYST/GB |
tomas@pop-os:~/go/src/github.com/mysteriumnetwork/node$ build/myst/myst proposals --location-type hosting --country UK
[INFO] No proposals found
```

Updates: https://github.com/mysteriumnetwork/node/issues/2837

### CODE REVIEW

File: cmd/commands/connection/util.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+func printProposal(p *contract.ProposalDTO) {
+	msg := fmt.Sprintf("| Identity: %s | Type: %s | Country: %s | Price: %s/min   %s/GB |",
+		p.ProviderID,
+		p.ServiceDefinition.LocationOriginate.NodeType,
+		p.ServiceDefinition.LocationOriginate.Country,
+		pricePerMinute(p.PaymentMethod),
+		pricePerGB(p.PaymentMethod),
+	)
+	fmt.Println(msg)
```
Comment (soffokl) [inline cmd/commands/connection/util.go:37]:
```suggestion
	fmt.Printf("| Identity: %s | Type: %s | Country: %s | Price: %s/min   %s/GB |\n",
		p.ProviderID,
		p.ServiceDefinition.LocationOriginate.NodeType,
		p.ServiceDefinition.LocationOriginate.Country,
		pricePerMinute(p.PaymentMethod),
		pricePerGB(p.PaymentMethod),
	)
```

File: cmd/commands/connection/util.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+func pricePerMinute(pm contract.PaymentMethodDTO) string {
+	if pm.Price.Amount == nil {
+		return "unknown"
```
Comment (soffokl) [inline cmd/commands/connection/util.go:42]: `unknown` price will confuse users. Is it free service or unsupported?

File: cmd/commands/connection/command.go [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+	flagLocationType = cli.StringFlag{
+		Name:  "location-type",
+		Usage: "Node location types to filter by eg.'hosting', 'residential', 'mobile' etc.",
+	}
+)
+
+const (
+	serviceWireguard = "wireguard"
+
+	gbInBytes = 1 << (10 * 3)
```
Comment (soffokl) [inline cmd/commands/connection/command.go:48]: We have it already defined in the https://github.com/mysteriumnetwork/node/blob/master/datasize/bitsize.go

**Review by soffokl — APPROVED** [review]

File: cmd/commands/connection/util.go (hunk 1 above)
Comment (tomasmik) [reply cmd/commands/connection/util.go:37]: Dont need the new line. We already do `Println`

File: cmd/commands/connection/util.go (hunk 2 above)
Comment (tomasmik) [reply cmd/commands/connection/util.go:42]: I had instances where Price.Amount was `nil` I dont know if it's free or unsupported, I'll have to look in to it.

File: cmd/commands/connection/util.go [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		ppm,
+		ppgb,
+	)
+}
+
+func pricePerMinute(pm contract.PaymentMethodDTO) (string, bool) {
+	s := time.Duration(pm.Rate.PerSeconds) * time.Second
+	min := new(big.Float).SetFloat64(s.Minutes())
+	if min.Cmp(big.NewFloat(0)) == 0 {
+		// If a node is missconfigured and somehow reports Rate as 0
```
Comment (soffokl) [inline cmd/commands/connection/util.go:54]: Some nodes can provide a service for free, does this prevent user from seen such nodes?

**Review by vkuznecovas — APPROVED** [review]

**Pull request merged**